### PR TITLE
QSP-1227 improve logging for invalid json

### PR DESCRIPTION
**Pull request by arshdeeptinna**
## Description

Currently no log is generated when keystore file is invalid. This change should create a log similar to the following
```
{"message": "Error in running node: Keystore file invalid. Replace with valid keystore file.", "threadName": "MainThread", "lineno": 202, "pathname": "/app/src/qsp_protocol_node/__main__.py", "logger": "__main__", "level": "error", "timestamp": "2019-07-25T20:15:23.220581Z", "exception": "Traceback (most recent call last):\n  File \"/app/src/qsp_protocol_node/__main__.py\", line 195, in <module>\n    os.environ['QSP_LOGGING_LEVEL']\n  File \"/app/src/qsp_protocol_node/__main__.py\", line 125, in setup\n    raise InvalidKeyStoreError(\"Keystore file invalid. Replace with valid keystore file.\")\nInvalidKeyStoreError: Keystore file invalid. Replace with valid keystore file."}
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have updated the version in src/qsp_protocol_node/config/config.py

### CODE REVIEW

File: src/qsp_protocol_node/__main__.py [hunk 1]
```diff
@@ -116,7 +112,22 @@ def setup(cls, env, yaml_file_name, log_level):
...
+            with open(keystore_file) as k:
+                keystore = load(k)
+            # Get account
+            account = Web3.toChecksumAddress('0x' + keystore['address'])
+
+        except KeyError:
+            pass
+
+        if not account:
+            raise InvalidKeyStoreError("Keystore file invalid. Replace with valid keystore file.")
```
Comment (alex-murashkin) [inline src/qsp_protocol_node/__main__.py]: Can we also print the location of the file and add a reference to the README on how to address the issue?

File: src/qsp_protocol_node/__main__.py [hunk 2]
```diff
@@ -178,18 +189,17 @@ def run(cls, eth_passphrase, eth_auth_token, sol_file):
 if __name__ == "__main__":
     logger = None
     try:
+        from log_streaming import get_logger
```
Comment (alex-murashkin) [inline src/qsp_protocol_node/__main__.py]: Are there any implications of this re-ordering?

File: bin/create-bundle [hunk 3]
```diff
@@ -48,7 +48,6 @@ cp -a "$QSP_ROOT/doc/." "$DIST_DIR/doc/"
 cp "$SCRIPT_DIR/stop-node" "$SCRIPT_DIR/common" "$DIST_DIR/bin"
 cp -a "$CONTRACTS_DIR/." "$DIST_DIR/resources/contracts"
 mkdir -p "$DIST_DIR/resources/keystore"
-echo "Add a valid key here" > "$DIST_DIR/resources/keystore/default.json"
```
Comment (alex-murashkin) [inline bin/create-bundle]: Do our current instructions in the operator's manual assume the file exists?

Comment (arshdeeptinna) [reply bin/create-bundle]: File is already copied and has `{"Error": "Replace with a valid keystore file"}`. This echo command is not needed

File: src/qsp_protocol_node/__main__.py (hunk 2 above)
Comment (arshdeeptinna) [reply src/qsp_protocol_node/__main__.py]: looks like there might be .. back to work

**Comment by arshdeeptinna** [comment]
@lpassos  addressed your comment

File: src/qsp_protocol_node/__main__.py [hunk 4]
```diff
@@ -145,7 +155,6 @@ def run(cls, eth_passphrase, eth_auth_token, sol_file):
 
         logger.debug("min_price_in_qsp: {0}".format(cfg.min_price_in_qsp))
         logger.debug("evt_polling: {0}".format(cfg.evt_polling))
-        logger.debug("audit contract address: {0}".format(cfg.audit_contract_address))
```
Comment (alex-murashkin) [inline src/qsp_protocol_node/__main__.py]: Can we keep this log message?

File: src/qsp_protocol_node/__main__.py [hunk 5]
```diff
@@ -176,24 +185,30 @@ def run(cls, eth_passphrase, eth_auth_token, sol_file):
...
+            raise setup_exception
 
+        Program.run(os.environ['QSP_ETH_PASSPHRASE'], os.environ['QSP_ETH_AUTH_TOKEN'], sol_file)
     except Exception as error:
-        if logger is not None:
+        if logger:
             logger.exception("Error in running node: {0}".format(str(error)))
         else:
             traceback.print_exc()
-        # A non-zero exit code is required to auto-restart
```
Comment (alex-murashkin) [inline src/qsp_protocol_node/__main__.py]: Should we keep this comment?

File: src/qsp_protocol_node/__main__.py (hunk 4 above)
Comment (lpassos) [reply src/qsp_protocol_node/__main__.py]: @alex-murashkin That line was duplicated.

File: src/qsp_protocol_node/__main__.py (hunk 5 above)
Comment (arshdeeptinna) [reply src/qsp_protocol_node/__main__.py]: fixed

**Pull request merged**